RealmSwift/RealmCollection.swift

```diff
@@ -85,7 +85,7 @@ public protocol _RealmMapValue {
     public mutating func next() -> Element? {
         let next = generatorBase.next()
         if let key = next as? Key,
-           let value = collection[key as AnyObject].map(dynamicBridgeCast) as? Value {
+            let value = collection[key as AnyObject].map(Value._rlmFromObjc(_:)), let value {
```

Comment:
Previous indentation was correct.

`RLMMapIterator` presumably needs to be changed as well (and tests added for that).

Reply:
This was not fixed.